cloudigrade/analyzer/tests/test_management_commands.py

```diff
@@ -5,39 +5,46 @@
 from io import StringIO
 from unittest.mock import patch
 
+from dateutil import tz
 from django.core.management import call_command
 from django.test import TestCase
 from django.utils.translation import gettext as _
 
-from account.management.commands.add_account import aws
-from util.tests import helper
+from account.models import Instance, InstanceEvent
+from account.tests import helper as account_helper
+from analyzer.management.commands.analyze_log import aws
+from util.tests import helper as util_helper
 
 
 class AnalyzeLogTest(TestCase):
     """Analyze Log management command test case."""
 
-    def test_command_output_success(self):
-        """Test processing a CloudTrail log."""
+    def setUp(self):
+        """Set up common variables for tests."""
+        self.mock_account_id = str(util_helper.generate_dummy_aws_account_id())
+        self.mock_arn = util_helper.generate_dummy_arn(self.mock_account_id)
+        self.mock_account = account_helper.generate_account(
+            self.mock_arn, self.mock_account_id,
+        )
+
+    def test_command_output_success_ec2_attributes_included(self):
+        """Test processing a CloudTrail log with all data included."""
         out = StringIO()
         mock_queue_url = 'https://sqs.queue.url'
         mock_receipt_handle = str(uuid.uuid4())
-        mock_account_id = str(helper.generate_dummy_aws_account_id())
         mock_instance_id = str(uuid.uuid4())
         mock_region = 'us-east-1'
         mock_event_type = 'RunInstances'
-        mock_occured_at = datetime.datetime.utcnow().isoformat()
+        now = datetime.datetime.utcnow()
+        # Work around for sqlite handling of microseconds
```

Comment:
Oooh… I don't know if I've experienced or noticed this implied sqlite microsecond problem. What specifically did this work around?

Reply:
Specifically that `utcnow()` returns a datetime with microseconds, and when you read back the entry from the DB it cuts off at seconds. I assumed/thought it was sqlite, but maybe Django? My datetimes were definitely not equal going in and coming out though.